source/_integrations/vodafone_station.markdown

```diff
@@ -21,6 +22,7 @@ The Vodafone Station integration allows you to control your [Vodafone Station](h
 There is support for the following platform types within Home Assistant:
 
 - **Device tracker** - presence detection by looking at connected devices.
+- **Sensor** - external IP address, uptime, firmware, resources and network monitors.
 {% include integrations/config_flow.md %}
 
 ## Configuration
```

Comment:
General notice: the config flow include also has a Configuration header, so now the docs have 2 Configuration headers :)

Reply:
Thx, cleaned up